src/easylink/pipeline_graph.py

```diff
@@ -191,6 +250,8 @@ def _merge_combined_implementations(self, config: Config) -> None:
             ) = self._get_combined_slots_and_edges(combined_implementation, nodes_to_merge)
 
             # Create new implementation
+            # TODO: If any of the implemented steps is an EmbarrassinglyParallelStep,
+            # raise immediately (for now - will be a new feature/task)
```

Comment:
Feels like we should actually do this TODO before merging, since it's easy

Reply:
ok, implemented - thank god we implemented the custom schema support!